systers_portal/static/js/pin.js

```diff
@@ -0,0 +1,51 @@
+function getCookie(name) {
+	var cookieValue = null;
+	if (document.cookie && document.cookie !== "") {
+		var cookies = document.cookie.split(";");
+		cookies.forEach(function(cookie){
+			cookie = jQuery.trim(cookie);
+                    // Does this cookie string begin with the name we want?
+                    inner:if (cookie.substring(0, name.length + 1) === (name + "=")) {
+                    	cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
+                    	break inner;
+                    }
+                });
+	}
+	return cookieValue;
+}
+$("#pin").click(function() {
+	$(this).hide();
+	$("#remove").show();
+	let csrftoken = getCookie("csrftoken")
+	let Data = {
+		"csrfmiddlewaretoken": csrftoken,
+	};
+	$.ajax({
+		type: "POST",
+		url: "pin/",
+		data: Data,
+		success(a){
+			console.log("Pinned")
+		},
+		dataType:"json",
+	});
+});
+
+$("#remove").click(function() {
+	$("#pin").show();
+	$(this).hide();
+	$("#pin").show();
+	let csrftoken = getCookie("csrftoken")
+	let Data = {
+		"csrfmiddlewaretoken": csrftoken,
```

Comment:
Just the CSRF token and nothing else?

Reply:
@SanketDG he is using the resource slug to get the resource. Atleast that's what it seems from the RemovePinView